pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -164,32 +165,53 @@ class EmuFreeBackend(PasqalEmulator):
     default_config = DEFAULT_CONFIG_EMU_FREE
 
 
-class EmuMPSBackend(RemoteBackend):
+class RemoteEmulatorBackend(RemoteBackend, EmulatorBackend):
+    _device_type: DeviceTypeName
+
     def __init__(
         self,
         sequence: pulser.Sequence,
         connection: PasqalCloud,
-        config: EmulationConfig,
+        *,
+        config: EmulationConfig | None = None,
         mimic_qpu: bool = False,
     ) -> None:
-        super().__init__(
+        RemoteBackend.__init__(
+            self,
             sequence=sequence,
             connection=connection,
             mimic_qpu=mimic_qpu,
         )
-        if not isinstance(config, EmulationConfig):
-            raise TypeError(
-                "'config' must be an instance of 'EmulationConfig', "
-                f"not {type(config)}."
-            )
-        self._config = config
-        self._device_type = pasqal_cloud.DeviceTypeName.EMU_MPS
+        EmulatorBackend.__init__(
+            self,
+            sequence,
+            config=config,
+            mimic_qpu=mimic_qpu,
+        )
 
     def _submit_kwargs(self) -> dict[str, Any]:
         """Keyword arguments given to any call to RemoteConnection.submit()."""
         return dict(
             batch_id=self._batch_id,
+            # To be deleted once this PR is released: https://github.com/pasqal-io/Pulser/pull/888
             mimic_qpu=self._mimic_qpu,
             backend_configuration=self._config,
             device_type=self._device_type,
         )
+
+
+class EmuMPSBackend(RemoteEmulatorBackend):
+    default_config = EmulationConfig(observables=[BitStrings()])
```

Comment:
Let's add the parameters Anton suggested:
```suggestion
    default_config = EmulationConfig(
        observables=[BitStrings()],
        num_gpus_to_use=1,
        autosave_dt=float("inf"),
        optimize_qubit_ordering=True,
        )
```